boards/arm/stm32h750b_dk/stm32h750b_dk.dts

```diff
@@ -18,6 +18,15 @@
 		zephyr,shell-uart = &usart3;
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
+		zephyr,flash-controller = &mt25ql512ab1;
+	};
+
+	sdram1: sdram@c0000000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		device_type = "memory";
+		reg = <0xc0000000 DT_SIZE_M(32)>; /* 128Mbit */
```

Comment:
I am confused how the test could pass?
Also, shouldn't this be sdram2 at 0xd0000000 instead, as per BSP [here](https://github.com/STMicroelectronics/stm32h750b-dk-bsp/blob/main/stm32h750b_discovery_sdram.h#L62)?

```suggestion
		reg = <0xc0000000 DT_SIZE_M(16)>; /* 128Mbit */
```

Reply:
Should indeed be 16 Mb = 128 Mbit, this was an oversight on my side. Not sure how this influences tha passing or not passing of the tests.